core/src/main/java/org/zenoss/zep/impl/TriggerPlugin.java

```diff
@@ -877,9 +877,10 @@ protected synchronized void processSpool(long processCutoffTime) {
                 }
 
                 int repeatInterval = trSub.getRepeatSeconds();
+                boolean onlySendInitial = trSub.getSendInitialOccurrence();
 
                 // Schedule the next repeat
-                if (repeatInterval > 0) {
+                if (!onlySendInitial && repeatInterval > 0) {
```

Comment:
I don't think this assumption is correct. Repetition and sending notification only on the initial occurrence are not exclusive. For a trigger that happens very frequently (like a specific event coming) it is possible to send the notification for the first occurrence only and still get a reminder every x seconds.